tests/Application/templates/bundles/SyliusShopBundle/_scripts.html.twig

```diff
@@ -1 +1,40 @@
 {{ encore_entry_script_tags('shop-entry', null, 'shop') }}
+
+{# To be pci dss compliant, soge commerce API states that SRI should be integrated #}
+{# See https://sogecommerce.societegenerale.eu/doc/fr-FR/rest/V4.0/javascript/redirection/sri.html #}
+
+<script type="text/javascript"
+        src="https://static-sogecommerce.societegenerale.eu/static/js/krypton-client/V4.0/stable/kr-payment-form.min.js"
+        integrity="sha384-FIqRg5Z3CArcz1UvV7osFqEHYW01awqNCTwwTYoL6lhBgo7+3t9yswE3vO93wOGv"
```

Comment:
Is there any way to have an env variable to handle version ? Maybe we prefer to keep control and make a new release if a new version is released but it seems like a lot of work to me.